tests/src/common/dispatcher_wrapper.rs

```diff
@@ -4,27 +4,26 @@ use rewards_dispatcher::{
     msg::{ExecuteMsg, InstantiateMsg, QueryMsg, SudoMsg},
     state::reward_scale::{Bar, RewardScale, TotalValueLocked},
 };
-use sdk::{cosmwasm_std::Addr, cw_multi_test::Executor};
+use sdk::cosmwasm_std::Addr;
 
-use crate::common::{ContractWrapper, MockApp};
+use super::{test_case::WrappedApp, ContractWrapper, ADMIN};
 
-use super::ADMIN;
-
-pub struct DispatcherWrapper {
+pub(crate) struct DispatcherWrapper {
     contract_wrapper: Box<DispatcherContractWrapper>,
```

Comment:
why we use Box here?

Reply:
`App::store_code` requires `Box<dyn Contract>` so that is consumed on instantiation. To be honest, it doesn't seem to matter where we put it, because it will be there no matter what.